encryption_works.md

```diff
@@ -254,21 +254,23 @@ In addition to protecting PGP secret keys, you also need to choose good passphra
 
 ### Software
 
-To install GPG, Windows users can download [Gpg4win](http://www.gpg4win.org/), and Mac OS X users can download [GPGTools](https://gpgtools.org/). If you run GNU/Linux you should already have GPG installed. GPG is a command line program, but there's software that interfaces with email clients that makes it much easier to use.
+To install GPG, Windows users can install [Gpg4win](http://www.gpg4win.org) and Mac OS X users can download [GPGTools](https://gpgtools.org). You'll also have to download a desktop e-mail client to use PGP correctly. This is because PGP does not yet function satisfactorily in a web browser. Freedom of the Press recommends using [Thunderbird](https://www.mozilla.org/en-US/thunderbird) as your desktop e-mail client with the [Enigmail](https://www.enigmail.net/home/index.php) add-on. 
```

Comment:
I don't think if we should delete references to Linux. After all, we are encouraging people to use open source throughout this guide, so making it clear there is an open source version of an operating system is important.